### ISSUE TITLE
Using keyboard to jump between widget areas is difficult

### ISSUE THREAD

**Issue by mapk** [WordPress/gutenberg#25877]
### Problem
Using the keyboard to jump between widget areas is difficult. I tried using `TAB` and/or using `Arrow Down` key, but it appears I have to jump through all the blocks before I can get to the next widget area. Notice that as I finally jumped to other areas, I couldn't get the accordion to open for me.

### Solution
I recommend that when the area is selected, using `TAB` jumps to the next widget area. If I want to jump into that particular widget area, using the `Arrow Down` to do so makes sense to me, or click `Enter` to jump to the first block in the area works too. But I'm open to any correction here from the Accessibility Team.

### Example of problem
![keyboard](https://user-images.githubusercontent.com/617986/95266708-92b3dc00-07e8-11eb-8ec4-a59cdaacd4cb.gif)

**Comment by mapk** [comment]
Related to https://github.com/WordPress/gutenberg/issues/25766

**Comment by talldan** [comment]
I did some testing, and it looks like I could use the down arrow to move to the next widget area, but I would expect it to expand (if collapsed) when pressing `space`.

**Comment by talldan** [comment]
When testing https://github.com/WordPress/gutenberg/issues/25880, I noticed that when a widget area is focused, you can use tab to highlight the panel at which point space can be used to open the panel.  As mentioned, the number of tab stops is potentially confusing. Usually for blocks the first focusable element in the block receives focus, which I would expect to be the same in a widget area..

**Comment by enriquesanchez** [comment]
I also ran into this while testing and think we should replicate the behavior from the editor. When a whole widget are is selected, and whether it's expanded or collapsed, I should be able to navigate through them with <kbd>TAB</kbd> and also <kbd>Up</kbd> and <kbd>Down</kbd> arrows.

**Comment by draganescu** [comment]
Related #25877.

**Comment by tellthemachines** [comment]
Currently, arrow down moves into the widget area when it's opened, and to the next area when it's closed. But it's now easier to open/close widget areas by pressing enter or space when the header is focused.

Tab moves to the sidebar, which is the same behaviour as in the post editor. 

Is there any further action required for this issue?

**Comment by draganescu** [comment]
I think we're ok!
